homeassistant/requirements.py

```diff
@@ -85,15 +85,20 @@ def async_clear_install_history(hass: HomeAssistant) -> None:
     _async_get_manager(hass).install_failure_history.clear()
 
 
-def pip_kwargs(config_dir: str | None) -> dict[str, Any]:
-    """Return keyword arguments for PIP install."""
+async def async_pip_kwargs(config_dir: str | None) -> dict[str, Any]:
+    """Return keyword arguments for PIP install.
+
+    This function is a coroutine.
+    """
```

Comment:
```suggestion
    """Return keyword arguments for PIP install."""
```
Not needed, since we can see that it is a coro from the function signature.